src/App.tsx

```diff
@@ -517,7 +580,38 @@ function App() {
 
   return (
     <div className="d-flex py-3 bg-light" style={{minHeight: '100vh'}}>
+      <button
+        type="button"
+        className="btn btn-secondary"
+        onClick={() => setSettingsOpen(true)}
+        style={{position: 'absolute', top: 12, right: 12}}
+      >
+        <span className="fa fa-gear" />
+      </button>
       <RouterProvider router={router} />
+      <Modal isOpen={settingsOpen} toggle={() => setSettingsOpen(false)}>
+        <ModalHeader toggle={() => setSettingsOpen(false)}>Settings</ModalHeader>
+        <ModalBody>
+          <form>
+            <div className="form-group">
+              <label htmlFor="ordering">Player ordering</label>
+              <select
+                className="form-control"
```

Comment:
Should be `form-select` to give proper look (down arrow at the right) – will fix.